frontend/pnpm-lock.yaml

```diff
@@ -1,289 +1,329 @@
-lockfileVersion: 5.4
+lockfileVersion: '6.0'
 
 overrides:
   '@automerge/automerge-wasm': file:deps/automerge-wasm-0.1.25.tgz
   '@automerge/automerge': file:deps/automerge-2.0.2.tgz
 
-specifiers:
-  '@audapolis/webvtt-writer': ^1.0.6
-  '@automerge/automerge': file:deps/automerge-2.0.2.tgz
-  '@automerge/automerge-wasm': file:deps/automerge-wasm-0.1.25.tgz
-  '@fontsource/inter': ^4.5.15
-  '@popperjs/core': ^2.11.7
-  '@tailwindcss/forms': ^0.5.3
-  '@types/react': ^18.0.28
-  '@types/react-dom': ^18.0.11
-  '@types/wavesurfer.js': ^6.0.3
-  '@typescript-eslint/eslint-plugin': ^5.54.0
-  '@typescript-eslint/parser': ^5.54.0
-  autoprefixer: ^10.4.13
-  clsx: ^1.2.1
-  eslint: ^8.35.0
-  eslint-config-prettier: ^8.6.0
-  eslint-plugin-import: ^2.27.5
-  eslint-plugin-react: ^7.32.2
-  lib0: ^0.2.63
-  npm-run-all: ^4.1.5
-  openapi-typescript: ^6.2.0
-  openapi-typescript-fetch: github:transcribee/openapi-typescript-fetch#content-type
-  postcss: ^8.4.21
-  prettier: ^2.8.4
-  react: ^18.2.0
-  react-base16-styling: ^0.9.1
-  react-dom: ^18.2.0
-  react-hook-form: ^7.43.5
-  react-icons: ^4.8.0
-  react-json-tree: ^0.18.0
-  react-popper: ^2.3.0
-  slate: ^0.94.0
-  slate-automerge-doc: github:transcribee/slate-automerge-doc
-  slate-react: ^0.94.0
-  swr: ^2.1.0
-  tailwindcss: ^3.2.7
-  typescript: ^4.9.5
-  vite: ^4.1.4
-  vite-plugin-top-level-await: ^1.3.0
-  vite-plugin-wasm: ^3.2.2
-  wavesurfer-react: ^2.2.2
-  wavesurfer.js: ^6.6.2
-  wouter: ^2.10.0
-
 dependencies:
-  '@audapolis/webvtt-writer': 1.0.6
-  '@automerge/automerge': file:deps/automerge-2.0.2.tgz
-  '@automerge/automerge-wasm': file:deps/automerge-wasm-0.1.25.tgz
-  '@fontsource/inter': 4.5.15
-  '@popperjs/core': 2.11.7
-  clsx: 1.2.1
-  lib0: 0.2.63
-  openapi-typescript-fetch: github.com/transcribee/openapi-typescript-fetch/1fcb5d7f9f0a2a4d26eec58e1a0a9d8be1655190
-  react: 18.2.0
-  react-base16-styling: 0.9.1
-  react-dom: 18.2.0_react@18.2.0
-  react-hook-form: 7.43.5_react@18.2.0
-  react-icons: 4.8.0_react@18.2.0
-  react-json-tree: 0.18.0_pmekkgnqduwlme35zpnqhenc34
-  react-popper: 2.3.0_d52dtujcogewjimhlbpfze4rs4
-  slate: 0.94.0
-  slate-automerge-doc: github.com/transcribee/slate-automerge-doc/b85a3ffb7c7a53c63ce4a01c2da0808d7c647a3d_slate@0.94.0
-  slate-react: 0.94.0_cvvhpwyf242mehno32vehwo6xe
-  swr: 2.1.0_react@18.2.0
-  tailwindcss: 3.2.7_postcss@8.4.21
-  vite-plugin-top-level-await: 1.3.0_vite@4.1.4
-  vite-plugin-wasm: 3.2.2_vite@4.1.4
-  wavesurfer-react: 2.2.2_wavesurfer.js@6.6.2
-  wavesurfer.js: 6.6.2
-  wouter: 2.10.0_react@18.2.0
+  '@audapolis/webvtt-writer':
+    specifier: ^1.0.6
+    version: 1.0.6
+  '@automerge/automerge':
+    specifier: file:/Users/anuejn/Developer/transcribee/frontend/deps/automerge-2.0.2.tgz
+    version: file:deps/automerge-2.0.2.tgz
+  '@automerge/automerge-wasm':
+    specifier: file:/Users/anuejn/Developer/transcribee/frontend/deps/automerge-wasm-0.1.25.tgz
+    version: file:deps/automerge-wasm-0.1.25.tgz
```

Comment:
Absolute paths don't seem like a good idea. Removing the overrides results in relative paths... Is this a pnpm bug?

Reply:
almost certainly

Reply:
it works with absolute paths but not with relative ones

Reply:
I think we need to use `pnpm install --no-frozen-lockfile` as ugly as it is

Reply:
Why do we need the overrides anyway?

Reply:
because they depend on each other?

Reply:
I think we can now just switch to automerge@2.0.3, which should contain all fixes we need. This also means we can get rid of the tar.gz in the repo

Reply:
🥳